src/placement/jump_map.c

```diff
@@ -688,6 +712,9 @@ get_object_layout(struct pl_jump_map *jmap, uint32_t layout_ver, struct pl_obj_l
 				rc = remap_alloc_one(&remap_list, k, target, false, remap_grp_used);
 				if (rc)
 					D_GOTO(out, rc);
+			} else {
+				if (domain != NULL)
+					setbit(dom_cur_grp_real, domain - root);
```

Comment:
is it possible to set the bit inside get_target()?
those bitmap and setbit at multiple places seems hard to understand TBH.